<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrincipalWindow</class>
 <widget class="QMainWindow" name="PrincipalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mostrar_correo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Bienvenido: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="buscador">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingresa el ID del producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_buscar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_id">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_name">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="nombre_producto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cantidad_producto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cerrar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar sesi√≥n</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
